server/entities/comment.ts

```diff
@@ -34,6 +42,12 @@ export class Comment {
   @MaxLength(500)
   private message: string;
 
+  @CreateDateColumn({ type: 'timestamp' })
+  private createdAt!: Date;
```

Comment:
This would need migrations too because the schema changed :) You can also use the cli tool to automatically create the queries for you when u have changed the model: https://github.com/typeorm/typeorm/blob/master/docs/migrations.md#generating-migrations

But we can agree that after this PR we start really doing migrations (expection is the user seed for now) because in production we don't want to lose any data

Reply:
I believe generating these changes are already late for me now, yes? I'll definitely start making the generations in the future though!

Reply:
It is not late because this PR is not deployed but we can start from next pr :D